arlo-client/src/components/Audit/RoundManagement/index.tsx

```diff
@@ -0,0 +1,150 @@
+  const { jurisdictions } = meta!
+  const jurisdictionId = jurisdictions[0].id
+
+  const [rounds, setRounds] = useState<IRound[]>([])
+  useEffect(() => {
+    if (electionId && jurisdictionId) {
+      ;(async () => {
+        try {
+          const response: { rounds: IRound[] } | IErrorResponse = await api(
+            `/election/${electionId}/jurisdiction/${jurisdictionId}/round`
+          )
+          // checkAndToast left here for consistency and reference but not tested since it's vestigial
+          /* istanbul ignore next */
+          if (checkAndToast(response)) return
+          setRounds(response.rounds)
+        } catch (err) /* istanbul ignore next */ {
+          // TEST TODO
+          toast.error(err.message)
+        }
+      })()
+    }
+  }, [electionId, jurisdictionId, setRounds])
+
+  const [ballots, setBallots] = useState<IBallot[]>([])
+  useEffect(() => {
+    if (electionId && jurisdictionId && rounds.length) {
+      ;(async () => {
+        try {
+          const response: { ballots: IBallot[] } | IErrorResponse = await api(
+            `/election/${electionId}/jurisdiction/${jurisdictionId}/round/${rounds[rounds.length - 1].id}/ballot-draws`
+          )
+          // checkAndToast left here for consistency and reference but not tested since it's vestigial
+          /* istanbul ignore next */
+          if (checkAndToast(response)) return
+          setBallots(response.ballots)
+        } catch (err) /* istanbul ignore next */ {
+          // TEST TODO
+          toast.error(err.message)
+        }
+      })()
+    }
+  }, [electionId, jurisdictionId, rounds, setBallots])
+
+  // const [{ online }] = useAuditSettings(electionId!)
+
+  const [auditBoards, setAuditBoards] = useState<IAuditBoard[]>([])
+  const getAuditBoards = useCallback(async () => {
+    try {
+      const response:
+        | { auditBoards: IAuditBoard[] }
+        | IErrorResponse = await api(
+        `/election/${electionId}/jurisdiction/${jurisdictionId}/round/${rounds[rounds.length - 1].id}/audit-board`
+      )
+      // checkAndToast left here for consistency and reference but not tested since it's vestigial
+      /* istanbul ignore next */
+      if (checkAndToast(response)) return
+      setAuditBoards(response.auditBoards)
+    } catch (err) /* istanbul ignore next */ {
+      // TEST TODO
+      toast.error(err.message)
+    }
+  }, [electionId, jurisdictionId, rounds])
+  useEffect(() => {
+    if (electionId && jurisdictionId && rounds.length) {
+      getAuditBoards()
+    }
+  }, [electionId, jurisdictionId, rounds, getAuditBoards])
+
+  if (rounds.length > 0 && rounds[rounds.length - 1].isAuditComplete)
+    return (
+      <Wrapper className="single-page">
+        <H2Title>
+          Congratulations! Your Risk-Limiting Audit is now complete.
+        </H2Title>
+      </Wrapper>
+    )
+  return (
+    <Wrapper className="single-page left">
+      <H2Title>Round 1 Audit Board Setup</H2Title>
```

Comment:
I'm assuming this says round 1 for now and will be updated to the actual round number at some future point?